app/web/src/index.css

```diff
@@ -17,6 +17,29 @@
   body {
     font-family: 'Fira Sans', ui-sans-serif, system-ui, -apple-system, system-ui, "Segoe UI", "Helvetica Neue", Arial, "Noto Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
   }
+
+  /* https://www.a11yproject.com/posts/2013-01-11-how-to-hide-content/ */
```

Comment:
The SSR class in tailwind looks pretty similar, definitely not the same though.
https://tailwindcss.com/docs/screen-readers

Reply:
Ah should've know they'd have a utility for that